pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -42,6 +42,7 @@
     "System.Threading.ThreadPool": "4.3.0",
     "Microsoft.DiaSymReader.Native": "1.4.1",
     "Libuv": "1.9.1",
+    "runtime.native.System.Security.Cryptography.Apple": "4.3.1-servicing-25511-01",
```

Comment:
I notice that System.Net.Http and System.Diagnostics.DiagnosticSource are both frozen at an older build number.  Microsoft.NETCore.Runtime.CoreCLR was the last version bump (several days ago), but this package didn't exist at 25505-01.

Do I need to wait until we re-baseline 1.1 on 25511-01 or higher?